paymentsheet-example/src/androidTest/java/com/stripe/android/lpm/TestAmazonPay.kt

```diff
@@ -0,0 +1,32 @@
+package com.stripe.android.lpm
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.stripe.android.BaseLpmTest
+import com.stripe.android.test.core.Currency
+import org.junit.Ignore
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+internal class TestAmazonPay : BaseLpmTest() {
+    private val amazonPay = newUser.copy(
+        paymentMethod = lpmRepository.fromCode("amazon_pay")!!,
+        currency = Currency.USD,
+        merchantCountryCode = "US",
+    )
+
+    @Ignore("Complex authorization handling required")
```

Comment:
We should figure this out before we merge this. Happy to help with that!

Reply:
Ah, I noticed that the issue is in the Web view. Is this something that will be fixed in the near future?

Reply:
I'm not sure about that, it will be updated to Amazon's own page once they start to build it out, so I marked it as ignored for now